minerva/tools/calendar.py

```diff
@@ -6,73 +8,81 @@
 
 
 class Event(NamedTuple):
-  summary: str
-  description: str
-  meet_url: Optional[str]
-  start: datetime
-  end: datetime
-  recurrence_rule: Optional[icalendar.vRecur]
-
-  def __str__(self):
-    return f"""Event: {self.summary} ({self.start.astimezone(tz=timezone.utc)} - {self.end.astimezone(tz=timezone.utc)})
+    summary: str
```

Comment:
nit: we use 2-space indent in the project; can you please run `make lint-fix` to ensure that the code is formatted correctly?

Reply:
sure